test/vanilla/pytest.ini

```diff
@@ -1,3 +1,4 @@
 [pytest]
 usefixtures = testserver
-xfail_strict=true
\ No newline at end of file
+xfail_strict=true
+python_files=test_*.py
```

Comment:
Added this since, without the `_asnc` suffix, pytest was trying to discover Validation's async service client (it would also recurse into the sync client but no errors would be thrown). Here, I'm just telling pytest to discover test files that start with `test_`